tests/conda_env/specs/test_requirements.py

```diff
@@ -4,7 +4,7 @@
 
 import unittest
 
-from .. import utils
+from .. import support_file
```

Comment:
I don't know how this was working before! This only failed in the CI tests downstream!

Reply:
Uh, weird! Good catch!